server/views/pages/deleteDraftLocation/confirm.njk

```diff
@@ -0,0 +1,86 @@
+{% extends "../../partials/layout.njk" %}
```

Comment:
I think you should be able to avoid a lot of duplication by extending formStep instead and adding to the blocks with super

Reply:
+ res.locals.titleCaption